doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -51,7 +65,7 @@ B<optype> is a mask indicating which operations the control can be applied to.
 The control command is indicated in B<cmd> and any additional arguments in
 B<p1> and B<p2>.
 
-For B<cmd> = B<EVP_PKEY_CTRL_SET_MAC_KEY>, B<p1> is the length of the MAC key,
+For B<cmd> = B<EVP_PKEY_CTRL_SET_PRIV_KEY>, B<p1> is the length of the MAC key,
```

Comment:
"When cmd is EVP_PKEY..."  Not the semi-math expression.

Reply:
Done